### PR TITLE
[RPS-19] Build working Hippo Delivery app

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
Also cleaned up Authoring app a bit.

### CODE REVIEW

File: ansible/roles/service/hippo_authoring/tasks/configure.yml [hunk 1]
```diff
@@ -18,6 +16,12 @@
     - conf/repository.xml
     - conf/tomcat-users.xml
 
+- name: Ensure ROOT folder is deleted
```
Comment (ivorc) [inline ansible/roles/service/hippo_authoring/tasks/configure.yml]: Why? Is it worth expanding this description a little bit?

File: ansible/roles/service/hippo_delivery/templates/sudoers.d/hippo_delivery.j2 [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{{ hippo_delivery.user }} ALL= NOPASSWD: /bin/systemctl start {{ hippo_delivery.user }}
```
Comment (ivorc) [inline ansible/roles/service/hippo_delivery/templates/sudoers.d/hippo_delivery.j2:2]: Suggest a comment "The service name is by definition the same as the user name", as discussed.

**Review by ivorc — APPROVED** [review]
Looks good.

File: ansible/roles/service/hippo_authoring/tasks/configure.yml (hunk 1 above)
Comment (wojtek-oledzki) [reply ansible/roles/service/hippo_authoring/tasks/configure.yml]: I will add comment with explanation

**Comment by wojtek-oledzki** [comment]
@ivorc review changes applied

**Pull request merged**